scripts/utils/price_utils.js

```diff
@@ -5,6 +5,13 @@ const BN = require("bn.js")
 const { decodeOrders, getUnitPrice } = require("@gnosis.pm/dex-contracts")
 const { Fraction } = require("@gnosis.pm/dex-contracts")
 
+const { TEN } = require("./constants")
+const { toErc20Units } = require("./printing_tools")
+
+/**
```

Comment:
Is this comment needed?

Reply:
I checked, having this comment removes a linting warning. When checking, I noticed we have a lot of linting warning already, mostly because of the JSDoc checking (see [here](https://travis-ci.com/github/gnosis/dex-liquidity-provision/builds/173123954#L217)). Maybe @bh2smith knows what's going on with the warnings?

Reply:
Yea. I imposed forced JSDocs in the entire project. You might recall reviewing and approving it in #269

Reply:
This is absolutely needed in order to import the TokenObject type if it is being used here. All this JSdoc crap will go away when we move to TS.